src/routes/locationSearchPage/LocationSearchPage.js

```diff
@@ -24,10 +24,8 @@ const StyledText = styled.h3`
     text-align: center;
 `
 
-// TODO: Delete this. This is only a temporary solution to the handle click
-//   callback.
 function handleClick() {
```

Comment:
if this is not doing anything, can you delete this?

Reply:
deleted